lib/Feature/FeatureModelWriter.cpp

```diff
@@ -12,7 +12,8 @@
 
 namespace vara::feature {
 
-static constexpr xmlChar IndentationString[] = "  ";
+static constexpr xmlChar INDENTATIONSTRING[] = "  ";
+static constexpr char ENCODING[] = "UTF-8";
 
 int FeatureModelXmlWriter::writeFeatureModel(std::string Path) {
   int Rc;
```

Comment:
```suggestion
  int RC;
```
Just a small nit, calling the error variable always `RC` would better align it with the check macro `CHECK_RC`